src/lib/config.ts

```diff
@@ -1334,15 +1335,14 @@ export function mergeDeep(o1, o2) {
     const r = Array.isArray(o1) ? o1.slice() : Object.create(o1)
     Object.assign(r, o1, o2)
```

Comment:
I think the only change required might be
```suggestion
    Object.assign(r, o1, Object.create(o2))
```

Reply:
Mh, not quite... This should probably use [window.structuredClone](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) instead of `Object.create`.

Reply:
The structuredClone is recursive, but we don't need a recursive copy in mergeDeep.
We have to recursively merge two config, so the mergeDeep must be recursive.
Therefore, add another recursive copy in mergeDeep seem to be redundant.

Reply:
Yes, mergeDeep should probably be changed so that `structuredClone` is called once on `o1` and `o2` and then a recursive merge is performed without any copying/object creation.